api/routes/userRouter.js

```diff
@@ -0,0 +1,7 @@
+const express = require("express");
+const router = express.Router();
+const userController = require("../controllers/userController");
```

Comment:
모듈 구분을 통해 구조 파악이 용이하도록 라인브레이킹을 해주면 좋을 것 같습니다.
```suggestion
const express = require("express");
const router = express.Router();

const userController = require("../controllers/userController");
```